src/styles/tools/_variebles.scss

```diff
@@ -0,0 +1,10 @@
+
+$light-gray: #616070;
+$black: #060b35;
+$light-blue: #00acdc;
+$font-size-medium: 12px;
+$star-size: 16px;
+$card-width: 200px;
+$right-gap: 50px;
+$card-elem: 4;
```

Comment:
just type 4 in your calculations. it's useless variable

Reply:
@TheVovchik, no 4 in calculations - is a magic number, better to have variable with amount of cards